### DISCUSSION TITLE
/echest, /craft and more

### DISCUSSION THREAD

**Discussion by GoncaloT** [GriefPrevention/GriefPrevention#1247] [Q&A] [answered]
Hello! I wan´t to enable the /echest, /craft and all this commands for all players in my server can you help me?
The people says that only Ops can use this command and i don´t know how to enable it to everyone
Thank you so much!

**Comment by SrBedrock** [comment, marked as answer]
I believe this question has nothing to do with the GriefPrevention plugin.
If you are using the EssentialsX plugin give your players the permission `essentials.workbench` for /craft and `essentials.enderchest` /echest

**Reply by GoncaloT** [reply]
thank you!
